src/group.coffee

```diff
@@ -55,6 +55,18 @@ class Group
   members: (group) =>
     sorted(@cache[group] or [])
 
+  info: (group) =>
+    if @cache[group].length > 0
+      arr = []
+      for m in @cache[group]
+        try
+          arr.push @robot.brain.data.users[m].name
+        catch err
+          arr.push m
```

Comment:
When would the above throw? And if this happens, can we do something more intelligent than just pushing m?